cms/core/db_router.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import DEFAULT_DB_ALIAS, router, transaction
+from django.db.models import Model
+
+
+class ReadReplicaRouter:  # pylint: disable=unused-argument,protected-access
```

Comment:
suggestion: link to the Django docs. e.g. https://docs.djangoproject.com/en/5.1/topics/db/multi-db/#automatic-database-routing